src/status_im/chat/models/reactions.cljs

```diff
@@ -5,7 +5,8 @@
             [status-im.transport.message.protocol :as message.protocol]
             [utils.re-frame :as rf]
             [taoensso.timbre :as log]
-            [utils.transforms :as transforms]))
+            [utils.transforms :as transforms]
+            [status-im2.contexts.chat.messages.drawers.view :as drawers]))
```

Comment:
Is it okay to use view namespaces in `model/events` ?
CC: @flexsurfer

Reply:
Updated